src/main/java/org/reaktivity/nukleus/tls/internal/TlsNukleus.java

```diff
@@ -207,13 +206,37 @@ static SSLContext initContext(
             TrustManager[] trustManagers = null;
             if (trustStoreFile.exists())
             {
+                trustStoreExists = true;
                 KeyStore trustStore = KeyStore.getInstance(trustStoreType);
                 trustStore.load(new FileInputStream(trustStoreFile), trustStorePassword.toCharArray());
                 // TODO: TLS Alert Record, code 112 / scope trustStore to match routes?
                 TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(
                         TrustManagerFactory.getDefaultAlgorithm());
                 trustManagerFactory.init(trustStore);
                 trustManagers = trustManagerFactory.getTrustManagers();
+                long authorization = 1;
+
+                for(String alias : Collections.list(trustStore.aliases()))
+                {
+                    if (trustStore.isCertificateEntry(alias))
+                    {
+                        if (caMap.size() >= 56)
+                        {
+                            // more than 56 ca certs, cannot fit in 7 bytes
+                            return null;
```

Comment:
What if there are more than `56` certs, but we are only using `2` of them?
Can we populate the authorization map on demand based on the needs of `ROUTE` and `RESOLVE`?